2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +16,17 @@
   
 */
 
-function getSettlers() {}
+function getSettlers(voyagers) {
+
+  let settlers = [];
+
+  voyagers.forEach(voyager => {
+    if (voyager.charAt(0) === "A" && voyager.includes("family")) {
```

Comment:
great! You can also use the string method `.endsWith()` to be more specific than `.includes()`